src/Tasks/Copy.cs

```diff
@@ -324,41 +329,28 @@ private void LogDiagnostic(string message, params object[] messageArgs)
                 Log.LogMessage(MessageImportance.Normal, FileComment, sourceFilePath, destinationFilePath);
 
                 File.Copy(sourceFileState.Name, destinationFileState.Name, true);
```

Comment:
Given that the destination is always deleted, I think it would be a good idea to set the `overwrite` flag to `false` to protect us from accidental overwrites.

Reply:
Yes, it is probably better to fail in case someone gets in between than overwriting a link. I will update the PR.

Reply:
I do not believe it is necessary. Sometimes logic might change to not delete it, or new Copy task argument will be introduced, or some weird concurrent logic is relaying with last win. I would recommend to keep it as is, but if you decide to change it, it has to go under same ChangeWave or same conditions as deleting files.